services/rds/src/test/java/software/amazon/awssdk/services/rds/PresignRequestHandlerTest.java

```diff
@@ -141,7 +141,7 @@ public void testSourceRegionRemovedFromOriginalRequest() throws URISyntaxExcepti
     private SdkHttpFullRequest marshallRequest(CopyDBSnapshotRequest request) throws URISyntaxException {
         SdkHttpFullRequest marshalled = SdkHttpFullRequestAdapter.toHttpFullRequest(marshaller.marshall(request));
         return marshalled.toBuilder()
-                         .endpoint(new DefaultServiceEndpointBuilder(RDSClient.SERVICE_NAME, Protocol.HTTPS.toString())
+                         .endpoint(new DefaultServiceEndpointBuilder("rds", Protocol.HTTPS.toString())
```

Comment:
Hmm should this be using endpoint prefix? For rds and most services it's the same but for this usage it seems to make more sense.